protos/sigstore_common.proto

```diff
@@ -154,13 +154,13 @@ enum SubjectAlternativeNameType {
 }
 
 message SubjectAlternativeName {
-        SubjectAlternativeNameType type = 1;
+        SubjectAlternativeNameType type = 1 [(google.api.field_behavior) = REQUIRED];
         oneof identity {
                 // A regular expression describing the expected value for
                 // the SAN.
-                string regexp = 2;
+                string regexp = 2 [(google.api.field_behavior) = REQUIRED];
                 // The exact value to match against.
-                string value = 3;
+                string value = 3 [(google.api.field_behavior) = REQUIRED];
```

Comment:
I have never used `REQUIRED` with a oneof, but I assume this would force a a situation where exactly one field is always set?

Reply:
As far as I understand, there's no way to add "required" on a `oneof` itself, so I guess, the convention is "all the oneof fields should be marked with required" which would have a meaning like "`oneof` should have value".

Of course, the client won't be able to fill several fields in `oneof` at the same time.

WDYT?

Reply:
Just in case, there is already "all fields are required in `oneof`" in the proto: https://github.com/sigstore/protobuf-specs/blob/16541696de137c6281d66d075a4924d9bbd181ff/protos/sigstore_bundle.proto#L73-L81

Reply:
Nice!